hoot-core/src/main/cpp/hoot/core/conflate/matching/MatchFactory.h

```diff
@@ -93,20 +96,17 @@ class MatchFactory
 
   void reset() { _creators.clear(); }
 
-  /**
-   * Returns the number of candidate matches for all registered match creators.
-   *
-   * @return the match candidate count
-   */
-  long getMatchCandidateCount(const ConstOsmMapPtr& map, const geos::geom::Envelope& bounds);
-
```

Comment:
Was this meant to be removed?  Jenkins fails a couple of the unit tests that might have caused this.
```
Failure: N4hoot30MatchCandidateCountVisitorTestE::runHighwayMatchCandidateCountTest
src/test/cpp/hoot/core/visitors/MatchCandidateCountVisitorTest.cpp(109)   - Expected: 8
- Actual  : 0
.F
Failure: N4hoot30MatchCandidateCountVisitorTestE::runMultipleScriptMatchCreatorTest
src/test/cpp/hoot/core/visitors/MatchCandidateCountVisitorTest.cpp(195)   - Expected: 21
- Actual  : 0
```

Reply:
Yes, that was a definition with no implementation...will doublecheck that though.  I added two tests to MatchCandidateCountVisitorTest as part of this, so I wonder if they're affecting those two failing tests at all.  will check into that.